blueprints/ember-cli-blueprint-test-helpers/index.js

```diff
@@ -1,12 +1,62 @@
 var Promise = require('ember-cli/lib/ext/promise');
+var existsSync = require('exists-sync');
+var fs = require('fs-extra');
+var path = require('path');
+var merge = require('lodash/merge');
+var writeFile = Promise.denodeify(fs.outputFile);
 
 module.exports = {
   description: 'Installs dependencies for ember-cli-blueprint-test-helpers',
+  availableOptions: [
+    {
+      name: 'babel',
+      type: Boolean,
+      default: false
+    }
+  ],
   normalizeEntityName: function(){},
-  afterInstall: function() {
-    return Promise.all([
+  afterInstall: function(options) {
+    var afterInstallTasks = [
       this.insertIntoFile('./.npmignore', 'node-tests/'),
-      this.addPackageToProject('mocha', '^2.2.1'),
-    ]);
+      this.addPackageToProject('mocha', '^2.2.1')
+    ];
+    if (options.babel) {
+      afterInstallTasks.splice(0, 0,
+        this.insertIntoFile('./node-tests/mocha.opts', '--require babel-register\n--recursive'),
+        this.addPackagesToProject([
+          {name: 'babel-plugin-transform-es2015-arrow-functions', target: '^6.5.2'},
+          {name: 'babel-plugin-transform-es2015-shorthand-properties', target: '^6.5.0'},
+          {name: 'babel-register', target: '^6.7.2'}
+        ]),
```

Comment:
this might cause concurrency problems with the commands above. in other words: calling `addPackagesToProject` twice will *start* npm twice. what we would want to do though is wait until the first command has finished before starting the second one or do everything in one command.